scripts/survived.py

```
@@ -0,0 +1,79 @@
+No commit-message heuristic can catch that: the deletion happened inside a
+version-bump commit and the word "revert" appears nowhere. So this asks the
+only question that actually settles it: are the lines the PR added still in
+main today?
+
+Usage: survived.py <ref> <pr-number> [<pr-number> ...]
+"""
+import re, subprocess, sys
+
+
+def git(*args):
+    return subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)